src/engraving/libmscore/chordrest.cpp

```diff
@@ -519,16 +519,17 @@ Element* ChordRest::drop(EditData& data)
 
     case ElementType::NOTE: {
         Note* note = toNote(e);
-        NoteVal nval;
-        nval.pitch = note->pitch();
-        nval.tpc1 = note->tpc1();
-        nval.headGroup = note->headGroup();
-        nval.fret = note->fret();
-        nval.string = note->string();
-        score()->setNoteRest(segment(), track(), nval, ticks(), Direction::AUTO);
-        delete e;
+        Segment* seg = segment();
+        score()->undoRemoveElement(this);
+        Chord* chord = new Chord(score());
+        chord->setTrack(track());
+        chord->setDurationType(durationType());
+        chord->setTicks(ticks());
+        chord->setTuplet(tuplet());
+        chord->add(note);
+        score()->undoAddCR(chord, seg->measure(), seg->tick());
```

Comment:
existing method failed to transfer all properties and didn't work with tuplets etc.